usr/lib/cca_stdll/cca_stdll.h

```diff
@@ -33,7 +33,7 @@
 
 #define CCA_PRIVATE_KEY_NAME_SIZE       64
 #define CCA_REGENERATION_DATA_SIZE      64
-#define CCA_KEY_TOKEN_SIZE              2500
+#define CCA_KEY_TOKEN_SIZE              8000
```

Comment:
Does this new size of 8000 bytes cover the rounded up max key token length of 7648 bytes of a CRYSTALS-Dilithium (8,7 Round 2) QSA token?

Reply:
The CCA doc mentions the following for parameter key_token_length on the PKA Key Token Build (CSNDPKB) verb (and possibly at other places):

_On input, a size of 8000 bytes is sufficient to contain the largest key_token created._

Thats basically where this length comes from.